spi/src/main/java/org/wildfly/swarm/spi/api/SwarmProperties.java

```diff
@@ -58,6 +58,17 @@
     String HTTPS_PORT = "swarm.https.port";
 
     /**
+     * If true, generates a self-signed certificate for development purposes
+     */
+    String HTTPS_GENERATE_SELF_SIGNED_CERTIFICATE = "swarm.https.certificate.generate";
```

Comment:
Could this also become part of the UndertowFraction API instead of using properties? 

i.e. `UndertowFraction#generateCert(bool)`